sdk/sdk-core/aar/src/main/java/mil/emp3/core/mapgridlines/coordinates/UTMCoordinate.java

```diff
@@ -0,0 +1,726 @@
+package mil.emp3.core.mapgridlines.coordinates;
```

Comment:
Since a number classes in this review are ported from some NASA example, it would be good to put a source information on top of those classes, for future maintenance purposes.